game/src/main/scala/io/aigar/model/Team.scala

```diff
@@ -53,6 +53,19 @@ object TeamDAO extends TableQuery(new Teams(_)) {
     )
   }
 
+  def updateScore(db: Database, team_id: Int, value: Int): Unit ={
+    val sql = sqlu"""update TEAMS
+                     set score = score + ${value}
+                     where id = ${team_id}"""
```

Comment:
Well not that much, I have to do plain SQL query to be able to increase the current score with the value since  `team.score = team.score + value` is not supported by slick.